content/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services.md

```diff
@@ -10,59 +10,65 @@ toc: true
 weight: 400
 ---
 
-This guide explains how to use the NGINX Open Source or F5 NGINX Plus Ingress Controller for Kubernetes as the Ingress controller for a Kubernetes cluster hosted in the Amazon Elastic Kubernetes Service (EKS).
+This guide explains how to use NGINX Open Source or F5 NGINX Plus Ingress Controller for Amazon Elastic Kubernetes Services (EKS).
 
-**Note:** The instructions in this document apply to both the NGINX and NGINX Plus Ingress Controllers for Kubernetes. For ease of reading, the document refers to NGINX Plus only.
+**Note:** These instructions apply to both the NGINX and NGINX Plus Ingress Controllers for Kubernetes. For ease of reading, the document refers to NGINX Plus only.
```

Comment:
```suggestion
{{< note >}} These instructions apply to NGINX Ingress Controller with NGINX or NGINX Plus. For ease of reading, the document refers to NGINX Plus only. {{< /note >}}
```

We use Hugo shortcodes wherever possible, and the most common you'll see is this note one.